src/domain/Mypage/MenuSummaryCard/MenuSummaryCard.module.scss

```
@@ -5,16 +5,18 @@
   width: 100%;
   height: 100%;
   padding: 18px 20px;
+  border-radius: 24px;
   overflow: hidden;
-  border: 1.5px solid f.color(gray-300);
-  box-shadow: 2px 2px 0 rgba(0, 0, 0, 0.1);
-  border-radius: 14px;
-
+  border: 1.5px solid f.color(gray-100);
+  box-shadow: 0 6px 12px 0 rgba(0, 0, 0, 0.08);
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)